### PR TITLE
Add securityContext

### LINKED ISSUE #711: helm: Add securityContext
**Is your feature request related to a problem? Please describe.**
Run the operator from a user other than root

**Describe the solution you'd like to see**
Add securityContext(s) to deployment template https://kubernetes.io/docs/tasks/configure-pod-container/security-context/

### PR DESCRIPTION
**Pull request by adamantal**
| Q               | A      |
| --------------- | ------ |
| Bug fix?        | no |
| New feature?    | no |
| API breaks?     | no |
| Deprecations?   | no |
| Related tickets | fixes #711 |
| License         | Apache 2.0 |

### What's in this PR?

Add `securityContext` value to the helm values.

### Why?

To configure the operator pod's `securityContext` field. 

### Checklist

- [x] Implementation tested
- [x] Error handling code meets the [guideline](https://github.com/banzaicloud/developer-guide/blob/master/docs/coding-style/error-handling-guide.md)
- [x] Logging code meets the guideline

### CODE REVIEW

**Review by chrisgacsal — APPROVED** [review]

File: charts/kafka-operator/templates/operator-deployment-with-webhook.yaml [hunk 1]
```diff
@@ -125,6 +125,10 @@ spec:
       {{- if .Values.additionalVolumes }}
       {{- include "chart.additionalVolumes" . | nindent 8 }}
       {{- end }}
+      {{- if .Values.securityContext }}
+      securityContext:
```
Comment (R-omk) [inline charts/kafka-operator/templates/operator-deployment-with-webhook.yaml:129]: `spec.containers[*] `    also has own property `securityContext`

e.g. 

https://github.com/banzaicloud/logging-operator/blob/master/charts/logging-operator/templates/deployment.yaml#L48-L52

Comment (adamantal) [reply charts/kafka-operator/templates/operator-deployment-with-webhook.yaml:129]: You're right. I'm going to introduce a separate field for pod and container level `securityContext`.

**Review by stoader — APPROVED** [review]

**Review by hi-im-aren — APPROVED** [review]

**Comment by adamantal** [comment]
@R-omk could you please take a look at it, before we merge this?

**Comment by R-omk** [comment]
> @R-omk could you please take a look at it, before we merge this?

LGTM

**Review by chrisgacsal — APPROVED** [review]

**Pull request merged**